e2e/e2e_test.go

```diff
@@ -73,7 +73,7 @@ var _ = Describe("Whereabouts functionality", func() {
 			clientInfo, err = wbtestclient.NewClientInfo(config)
 			Expect(err).NotTo(HaveOccurred())
 
-			netAttachDef = macvlanNetworkWithWhereaboutsIPAMNetwork(testNetworkName, testNamespace, ipv4TestRange)
+			netAttachDef = macvlanNetworkWithWhereaboutsIPAMNetwork(testNetworkName, testNamespace, ipv4TestRange, []string{})
```

Comment:
I don't understand this commit - it does nothing. 

It changes the test code to accept another parameter, but parameter is always passed empty. 

Please squash the commit that makes use of this so I can understand how this is used.

Reply:
Done :+1: